### PR TITLE
Fix #825 - fix add packages regression 

### LINKED ISSUE #825: PF5: Error in the wizard when selecting packages
When selecting packages in the create image wizard, the wizard errors out.

https://github.com/user-attachments/assets/3f772c6d-eb01-4197-9eef-88157dc2508f

### PR DESCRIPTION
**Pull request by amirfefer**
# Description

The regression is caused due to a query selector that relay on old PF4 classes.

Fixes #825 

## Type of change

What is it?

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update

### CODE REVIEW

**Comment by amirfefer** [comment]
https://github.com/user-attachments/assets/a817c709-2f15-4aa6-b8f5-7a6c36fa3d01

**Comment by InsightsDroid** [comment]
## :exclamation: Outdated dependencies error
* Installed version of react-router-dom (5.3.4) is older than required version (6.0.0)
* Installed version of @redhat-cloud-services/frontend-components-config (4.7.3) is older than required version (6.0.0)

File: src/components/form/AdditionalCustomPackages.js [hunk 1]
```diff
@@ -168,19 +168,19 @@ const AdditionalCustomPackages = ({ defaultArch, ...props }) => {
     }
 
     const destination = document.querySelector(
-      `.pf-m-${scrollTo.pane} .pf-c-dual-list-selector__menu`
+      `.pf-m-${scrollTo.pane} .pf-v5-c-dual-list-selector__menu`
```
[the remark this answers is not in the document]
Comment (ezr-ondrej) [reply src/components/form/AdditionalCustomPackages.js:171]: agreed, thanks for going against the urge to improve the code here :orange_heart:

**Review by ezr-ondrej — APPROVED** [review]
Nice! Thanks :orange_heart:

**Pull request merged**